ledger/ledger-on-memory/src/app/scala/com/daml/ledger/on/memory/InMemoryLedgerFactory.scala

```diff
@@ -50,8 +52,8 @@ private[memory] class InMemoryLedgerFactory(dispatcher: Dispatcher[Index], state
     )
   }
 
-  override def ledgerConfig(config: Config[Unit]): LedgerConfiguration =
-    LedgerConfiguration.defaultLocalLedger
+  override def initialLedgerConfig(config: Config[Unit]): InitialLedgerConfiguration =
+    super.initialLedgerConfig(config).copy(delayBeforeSubmitting = Duration.ZERO)
```

Comment:
A few ledgers now have different values for the delays. Was this intended?

The config submission delay used to be 500ms for kvutils ledgers (or any other single-node ledger where the index database is not the same as the ledger database). The previous comment says this is because there is a slight delay between the ledger and the index, although I don't think this would affect the initial submission for ledger-on-memory.

Reply:
Whoops, you're right, this should be 500ms.